### DISCUSSION TITLE
Does this look like real GFAP staining in the bone marrow?

### DISCUSSION THREAD

**Discussion by gokujianghu** [IBEXImagingCommunity/ibex_imaging_knowledge_base#61] [Q&A] [answered]
![Picture1](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/assets/86030402/8386a1a9-1d05-41fe-ae83-acc12661c2d0)

GFAP (red), DAPI (white)

**Comment by radtkea** [comment, marked as answer]
Greetings Gokujianghu,

Thank you for providing your image. Your bone marrow section looks very nice!

I've never examined GFAP in the bone marrow, but without further details I would guess no. This does not look like real GFAP staining in the bone marrow. 

1. The brightest GFAP signal is in a relatively acellular region of your tissue (the red stripe with little DAPI+ nuclei). 
2. GFAP should label the cytoplasm of cells based on [GeneCard](https://www.genecards.org/cgi-bin/carddisp.pl?gene=GFAP).
3. This [excellent paper](https://www.nature.com/articles/nbt.4006) examined GFAP in the bone marrow. Their staining pattern looks very different from yours. The authors are angels!!! Check out Supplementary Table 1. They report antibodies that work and do not work! Check out Supplementary Table 3. They tell you the cellular location of various markers in the bone marrow. They also authored a very detailed [protocol](https://www.nature.com/articles/nmeth.4503). 
4. Ultimately, they recommended this [antibody](https://www.thermofisher.com/antibody/product/GFAP-Antibody-Polyclonal/PA1-10019). 

In general, when trying to evaluate whether an antibody works or doesn't:
1. I first look at reference images provided by the vendor or antibody search engines like [CiteAb](https://www.citeab.com/). 
2. I check the subcellular location of the protein (nuclear, membrane, cytosolic) using [GeneCard](https://www.genecards.org/cgi-bin/carddisp.pl?gene=GFAP) or [UniProt](https://www.uniprot.org/uniprotkb/P03995/entry). 
3. I check tissue and cellular distribution using [Human Protein Atlas](https://www.proteinatlas.org/humanproteome/tissue). 
4. I Google! "Bone marrow imaging GFAP". 
5. I also use a positive control or colocalization with a validated marker for the same cell type. Can you test in the brain tissue? This is what most people are doing based on the vendor validation data.

Best of luck and please keep us posted on your GFAP testing!
A

![image](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/assets/102126229/210c6a97-dc41-44d8-b31e-908e654b28cb)

![image](https://github.com/IBEXImagingCommunity/ibex_imaging_knowledge_base/assets/102126229/2bfd3687-fff3-45dd-8f97-94a62d2c5871)
